_posts/2016-11-22-files-cpp.md

```diff
@@ -0,0 +1,153 @@
+| Ubuntu 14.04       | GCC 4.8         | Mostly supported        | Partially supported     |
+| Ubuntu 16.04       | GCC 5.3         | Fully supported         | Fully supported§        |
+| Visual Studio 2013 | VC12‖           | Partially supported     | Mostly unsupported      |
+| Visual Studio 2015 | VC13‖           | Supported               | Partially supported     |
+| Visual Studio 2017 | VC15‖           | Supported               | Supported               |
+
+\* The software collections
+  [devtoolset-4](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-4/)
+  (GCC 5.2) and
+  [devtoolset-3](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-3/)
+  (GCC 4.9) packages provide updated compilers which can build OME
+  Files C++ using C++11
+
+† Probably equivalent to Clang/LLVM 3.6 (it's a proprietary fork, so
+  we don't know the details)
+
+‡ Probably equivalent to Clang/LLVM 3.7 (it's a proprietary fork, so
+  we don't know the details)
+
+§ Boost 1.58 provided by this release does not build OME Files with
+  C++14, but does build with C++11; see [trac
+  ticket](https://svn.boost.org/trac/boost/ticket/11285)
+
+‖ Microsoft Visual C++ lags significantly behind GCC and Clang/LLVM,
+  but VS2013 contains a useful subset of features, which are improved
+  upon significantly with VC2015 and later
+
+The support classifications above are broad; please see the following
+references for full feature coverage:
+
+- [GCC support](https://gcc.gnu.org/projects/cxx-status.html#cxx11)
+- [Clang support](http://clang.llvm.org/cxx_status.html)
+- [Visual Studio support](https://msdn.microsoft.com/en-us/library/hh567368.aspx)
+
+## Third-party dependencies
+
+All of our third-party dependencies have *supported* building with a
+C++11 or C++14 compiler for some time.  However, more recently some of
+our dependencies have begin to *require* it, or will require it in
+their next major release.
+
+- Qt 5.7 (released June 2016) [requires
+  C++11](http://blog.qt.io/blog/2016/06/16/qt-5-7-released/)
+- ICU 59 will (expected April 2017) is [expected to require
+  C++11](https://sourceforge.net/p/icu/mailman/message/35232691/)
+  (delayed from ICU 58 to 59)
+- Other libraries we are considering to use require C++11
+
+Once these libraries are adopted by Linux distributions, MacOS X
+homebrew, FreeBSD ports and other packaging systems, OME Files C++
+will no longer build on these systems.  While some of these are
+optional (Qt5), others such as ICU are required by Boost and
+Xerces-C++ for full Unicode support, which is needed by the OME-XML
+data model.  As a result, C++11 support will become a necessity for
+continued use of these libraries as the versions requiring C++11 enter
+mainstream use.
+
+## The benefits of C++11
+
+C++11 adds a number of useful features to the language, including:
+
+- auto type
+- decltype
+- defaulted and deleted functions
+- delegating constructors
+- forward-declared enums
+- initaliser lists
```

Comment:
It's not a typo.  Do we have the same rules for posts as we do for formal documentation?